PJ3T3_Postie/Core/Setting/View/ThemeView.swift

```diff
@@ -10,13 +10,14 @@ import SwiftUI
 struct ThemeView: View {
     @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
```

Comment:
이거 @Environment(\.dismiss) var dismiss로 뷰 닫을 수 있어서 이걸로 변경해주시면 될 것 같아요!!

Reply:
현진님이 말씀해주신대로 해봤습니다. @Environment(\.dismiss) var dismiss를 선언하고 버튼쪽에 dismiss() 를 달아봤는데 적용도 안되고, 'Accessing Environment<DismissAction>'s value outside of being installed on a View. This will always read the default value and will not update.' 이라는 보라색 에러가 떠서 일단 원복 했습니다. 좀 더 알아보고 적용시켜볼게요.

Reply:
<img width="720" alt="image" src="https://github.com/APP-iOS3rd/PJ3T3_Postie/assets/119300554/94a64407-9a2d-4916-b7f6-24887c3ef18d">

특이하네요. 전 이런식으로 했는데 오류도 없고 뷰도 작동합니다. 알겠습니다~ 큰 이슈 아니니 다음에 적용해도 될 것 같습니다.

Reply:
아니; 지금은 또 잘 되네요.. ㅋㅋㅋ 이것도 이대로 적용해서 Theme PR 에 적용하겠습니다